tools/docker/builder/openwrt/Dockerfile

```diff
@@ -44,12 +46,16 @@ RUN git clone "$OPENWRT_REPOSITORY" openwrt \
 
 WORKDIR /home/openwrt/openwrt
 
+COPY --chown=openwrt:openwrt configs/sdk_${SDK_CONFIG}_config /home/openwrt/sdk_config
+
 RUN if [ -n "$INTEL_FEED" ] ; then \
-	echo "src-git intel $INTEL_FEED" >> feeds.conf.default \
-	&& scripts/feeds update intel \
-	&& scripts/feeds install intel_mips ; \
-    fi ; \
-    echo "CONFIG_SDK=y" > .config \
+        echo "src-git intel $INTEL_FEED" >> feeds.conf.default \
+        && scripts/feeds update intel \
+        && scripts/feeds install intel_mips ; \
+    fi ;\
+    cp /home/openwrt/sdk_config .config \
+    && cat .config \
```

Comment:
but if we move it to after the `defconfig` won't it include *all* the configuration options? I wanted to avoid that and only show the options we set by default.

Reply:
It will. It allows to check if the options you specified were actually used (they won't be kept in the file if they were unused for any reason).
The reasoning is that you already know what the defaults are for a specific prplMesh hash (by looking at the content of the file in the repository for that hash), but it's harder to know what the resulting SDK configuration will be (it's not kept in the image).
It's not that important though, it's mostly useful when debugging the build of the image itself.